integration/test_orgs.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+
+import pytest
+
+from py42.exceptions import Py42ForbiddenError
+from py42.exceptions import Py42InternalServerError
+
+
+timestamp = str(int(datetime.now().timestamp()))
+new_org = "integration test org {}".format(timestamp)
+org_id = 2689
+org_uid = 890854247383106706
+
+
+def test_create_org(connection):
+    with pytest.raises(Py42ForbiddenError):
+        connection.orgs.create_org(new_org)
+
+
+def test_get_agent_full_disk_access_states(connection):
+    response = connection.orgs.get_agent_full_disk_access_states(org_id)
+    assert response.status_code == 200
+
+
+def test_get_by_id(connection):
+    response = connection.orgs.get_by_id(org_id)
+    assert response.status_code == 200
+
+
+def test_get_page(connection):
+    response = connection.orgs.get_page(1)
+    assert response.status_code == 200
+
+
+def test_get_agent_state(connection):
+    response = connection.orgs.get_agent_state(org_id, "fullDiskAccess")
+    assert response.status_code == 200
+
+
+def test_get_by_uid(connection):
+    response = connection.orgs.get_by_uid(org_uid)
+    assert response.status_code == 200
+
+
+def test_deactivate(connection):
```

Comment:
This is a good way to deal with this... But I am just wondering if we should have a special marker for the operations that we don't expect to work for the prod environment but that we can make work in the server environment, we could have real responses there...

Reply:
Yeah, this is pretty dependent on the role of the user running the tests. If you happen to pass in the credentials of an org_admin or higher, this deactivate (and the `create()` above will actually succeed). 

Hard-coding with a specific org_id/org_uid also means if we want to run against a different env we'll need to manually change the code. 

I wonder if the integration tests should assume an admin user (fail fast if not), and allow for the `create()` to succeed first, then use the ID of that org from the response for any future actions?

Reply:
- The reason it was not assumed that all the tests will be run with `administrator` like privileges is because for such special cases the needed communication takes place, so that we can cover all cases possible. Yesterday discussing with @unparalleled-js we talked about `pytest markers` and may be we can use that in this case as well. While executing the script appropriate markers will be specified  and the tests will be executed with apt credentials against the markers. Before going ahead with this , we need to make sure the automation pipeline will handle that.

- on hard coded ids,   in all other features, the "create resource" task is done in a fixture and that id is used in subsequent tests, same can be done here if all the subsequent tests are executed with a user with `administrator` privileges or we create a user that can then call the subsequent tests, both approaches have different trade-offs to consider.

- Now that we are discussing the special cases, even in archives there is a hard-coded path, either we invest in creating setup/environment that can be standalone or we can mark certain tests to be executed in specific environments.

Reply:
So looking at all other hardcoded values, user_id is the most common, this can be fixed by creating a fixture that creates user and that fixture can be used in all other tests. Still we won't be able to pass the administrative privileges to the new user. However some tests in archives, orgs and security-data might need environment setup or privileges.

Reply:
I was only referring to his actual name, not the user ids

Reply:
It is a matter of courtesy